### ISSUE TITLE
Not compatible with Node 10

### ISSUE THREAD

**Issue by psvehla** [psvehla/mask-tracker#29]
For some inexplicable reason, I decided to upgrade to Node 10. Everything broke.

**Comment by psvehla** [comment]
Instead of just downgrading back to 7 (which is what I've been using), I decided to get with the times and upgrade everything. Bad move. This led me down a rabbit hole, and the job kept getting bigger and bigger. Partly because I'd not updated the project for a very long time. I got `ionic serve` going again, but the tests were completely broken. Unfortunately, I found that the boilerplate for testing in Ionic has [completely changed](https://lathonez.com/2018/ionic-2-unit-testing/), which left me looking at a complete rewrite of the tests. This also left me thinking that while I was there I should upgrade to a modern Ionic project structure. So I decided to pull the pin, reverting the changes. Although I find the app useful, it seems noone else does, and I'm not interested in this project enough anymore to warrant the effort.
